### PR TITLE
[11.0][MIG] sale_procurement_group_by_line

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Superseed of https://github.com/OCA/sale-workflow/pull/673 to put the CI in green.

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

File: sale_procurement_group_by_line/model/sale.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    @api.multi
+    def _get_procurement_group_key(self):
+        """ Return a key with priority to be used to regroup lines in multiple
+        procurement groups
+
+        """
+        return 8, self.order_id.id
+
+    @api.multi
+    def _action_launch_procurement_rule(self):
```
Comment (bjeficent) [inline sale_procurement_group_by_line/model/sale.py:32]: I think that you should explain that this function overwrite the standard without calling ```super```

File: sale_procurement_group_by_line/README.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+On itself, this module does nothing it is a requirement for modules which
+needs to create procurement group per sale order line basis.
+
+
+Usage
+=====
+
+To use this module, you need to install e.g.
+**Sale Sourced By Line** (`sale_sourced_by_line`)
```
Comment (bjeficent) [inline sale_procurement_group_by_line/README.rst]: I think this module is actually deprecated.

**Review by AaronHForgeFlow — COMMENTED** [review]
Agree with @bjeficent  and also put the overwrite thing in the readme please.

File: sale_procurement_group_by_line/model/sale.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply sale_procurement_group_by_line/model/sale.py:32]: I updated the PR with calling `super`. It's not an overwrite anymore. 
Please, update review everybody!

**Review by bjeficent — APPROVED** [review]
LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Pull request merged**